src/client/src/services/analytics/__mocks__/_analytics.ts

```diff
@@ -0,0 +1,6 @@
+import * as analytics from "../analytics"
+import * as MOCK_ANALYTICS from "./analytics"
+
+const _analytics = analytics as unknown as typeof MOCK_ANALYTICS
+
+export { _analytics }
```

Comment:
can we call these exports `blahMock` .. you can leave the filename as _blah.ts, as calling it analyticsMock.ts would be even more confusing ... maybe call it analyticsMockAdapter.ts, as it is an _Adapter_ (of sorts)
for brevity, this would work ..
```
export const analyticsMock = analytics as unknown as typeof MOCK_ANALYTICS
```